docs/_api/compositecontroller.md

```diff
@@ -392,3 +394,90 @@ That is, you should [set only the fields that you care about](/api/apply/).
 Note that your webhook handler must return a response with a status code of `200`
 to be considered successful. Metacontroller will wait for a response for up to the
 amount defined in the [Webhook spec](/api/hook/#webhook).
+
+### Finalize Hook
```

Comment:
Doc section LGTM-ed. I may post several questions here (for inquiry / verification purpose) here later when reviewing the codes.